.ci/generate_last_mod_file.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+stack_data=()
+sample_data=()
+
+grab_stacks() {
+    echo "Grabbing last modified date for stacks"
+
+    # look through git tree for all devfiles under stacks/
+    for filename in $(git ls-tree -r --name-only HEAD); do
+        if [[ $filename == *"stacks/"*"/devfile"* ]]; then
```

Comment:
Should we use the version directories (or stack's root if no multi-version support is present) latest commit instead of the individual `devfile.yaml`? Thinking of when resources are modified too.

Reply:
Yeah good idea, let me look into how to do that for an entire directory instead of just a file with the git tree

Reply:
@michael-valdron my last commit should set the `last_commit` date as the last time any file in the version directory is changed (or the root if there is no multi versions)